shuffle/cli/src/account.rs

```diff
@@ -82,5 +108,30 @@ pub fn handle(project_dir: PathBuf, account_key_path: PathBuf) -> Result<()> {
     );
     println!("Public key: {}", new_account.public_key());
 
+    let account_filepath = &address_dir.join(time.as_secs().to_string() + ".address");
+
+    let mut file = fs::File::create(account_filepath)?;
+    file.write_all(new_account
+        .address()
+        .to_string()
+        .as_ref()
+    )?;
+
+    Ok(())
+}
+
+#[derive(Debug, StructOpt)]
+pub enum AccountCommand {
+    // Rotates dev.key with new private key and saves old key in /keys
+    #[structopt(about = "Creates new account with randomly generated private/public key")]
+    New,
+}
+
+pub fn handle_package_commands(cmd: AccountCommand) -> Result<()> {
+    match cmd {
+        AccountCommand::New => {
+            handle()?;
+        }
+    }
     Ok(())
 }
```

Comment:
We have 0 tests here. Add a test for the file creation part, and save the node interaction for a later date when we incorporate forge. Break out the handle method above into 2 conceptual parts.